diff --git a/node_modules/@celo-tools/celo-ethers-wrapper/build/main/lib/CeloProvider.js b/node_modules/@celo-tools/celo-ethers-wrapper/build/main/lib/CeloProvider.js
index b380d38..71912b5 100644
--- a/node_modules/@celo-tools/celo-ethers-wrapper/build/main/lib/CeloProvider.js
+++ b/node_modules/@celo-tools/celo-ethers-wrapper/build/main/lib/CeloProvider.js
@@ -12,6 +12,9 @@ class CeloProvider extends ethers_1.providers.JsonRpcProvider {
         blockFormat.gasLimit = () => ethers_1.BigNumber.from(0);
         blockFormat.nonce = () => "";
         blockFormat.difficulty = () => 0;
+
+        const blockWithTransactionsFormat = this.formatter.formats.blockWithTransactions;
+        blockWithTransactionsFormat.gasLimit = () => ethers_1.BigNumber.from(0);
     }
     /**
      * Override to parse transaction correctly
